AssetManagementSystem/Controllers/AssetsController.cs

```diff
@@ -215,6 +217,17 @@ public ActionResult Search(string licenseplate, int? location)
             List<Asset> assets = new List<Asset>(); // all the assets we find
 
             // were we given a licenseplate?
+
```

Comment:
This looks like testing code that will always throw an exception and generate a log record.

It is supposed to be here for some reason?